client/src/main/java/com/tigrisdata/db/client/TigrisAsyncDatabase.java

```diff
@@ -31,20 +29,13 @@ public interface TigrisAsyncDatabase {
   CompletableFuture<List<CollectionInfo>> listCollections();
 
   /**
-   * Creates or updates collection under current database.
+   * Applies the schema from collection models to current database
    *
-   * @param schemaDirectory directory with schema
+   * @param collectionModelTypes an array of collection model classes
    * @return future to the {@link ApplySchemasResponse}
    */
-  CompletableFuture<ApplySchemasResponse> applySchemas(File schemaDirectory);
-
-  /**
-   * Creates or updates collection under current database.
-   *
-   * @param collectionsSchemas list of {@link URL}
-   * @return future to the {@link ApplySchemasResponse}
-   */
-  CompletableFuture<ApplySchemasResponse> applySchemas(List<URL> collectionsSchemas);
+  CompletableFuture<ApplySchemasResponse> applySchemas(
```

Comment:
we should call this CreateOrUpdateCollections now?

Reply:
updated the method name.